bin/prestart.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# Refresh all config files in order
+#
+# This script is typically called once at the container start, but
+# it can be called manually if WP config details must be changed
+
+# The database and memcached config files are separate to avoid collisions
+# if their backends' onChange handlers are triggered simultaneously
+echo "******running onstart script*********"
+
+until [[ `curl -s ${CONSUL}:8500/v1/health/state/passing | grep mysql-primary`  ]]
+do
+  echo "mysql-primary not healthly...."
+  sleep 5
+done
+
+until [[ `curl -s ${CONSUL}:8500/v1/health/state/passing | grep nfs`  ]]
+do
+  echo "no healthly nfs server avaliable yet...."
+  sleep 5
+done
+
+echo "mysql-primary and nfs are now healthly, moving on..."
+
+/usr/local/bin/onchange_reload-db.sh
+/usr/local/bin/onchange_reload-memcached.sh
+
+# The WordPress config file
+consul-template \
+    -once \
+    -dedup \
+    -consul ${CONSUL}:8500 \
+    -template "/var/www/html/consul-templates/wp-config.php.ctmpl:/var/www/html/wp-config.php"
+
+if $(wp --allow-root core is-installed)
+then
+  echo "WP is installed"
+  # run update-db to ensure the database schema is up to date in case the WP version was upgraded in the Docker image
+  wp --allow-root core update-db
+else
+  echo "WP is NOT installed"
+  echo "installing now...."
+  wp --allow-root core install --url=$WORDPRESS_URL --title="$WORDPRESS_SITE_TITLE" --admin_user=$WORDPRESS_ADMIN_USER --admin_password=$WORDPRESS_ADMIN_PASSWORD --admin_email=$WORDPRESS_ADMIN_EMAIL --skip-email
+  # update siteurl to work with our directory structure
+  # wp option update for siteurl REQUIRES http://, need to determine will we handle that here
```

Comment:
When you dont' include the scheme in the --url flag in the 'wp core install' command, you get problems with WP not rendering links correctly. Right now I am including it in the env file, i think in the prestart script I can check to make sure it is there and insert it if not. My only reservation with that is knowing weather or not to do http or https, but since we aren't supporting https as of yet, we should just default to http, eh?